### PR TITLE
ExchangeServerHunting

### PR DESCRIPTION
**Pull request by shabeebpm**
Artifact to hunt for suspicious activity in a Microsoft Exchange Server involves identifying key areas and log files to monitor

### CODE REVIEW

**Comment by shabeebpm** [comment]
Could you please update the status? I am unable to publish it, please advise.

**Comment by scudette** [comment]
This artifact is not valid - did you test it?

File: content/exchange/artifacts/ExchangeServerHunting [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: ExchangeServerHunting
+type: COMPOUND
```
Comment (scudette) [inline content/exchange/artifacts/ExchangeServerHunting:2]: No such type

File: content/exchange/artifacts/ExchangeServerHunting [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+description: >
+  Artifact for hunting suspicious activity on Microsoft Exchange Servers,
+  focusing on web shells, suspicious PowerShell commands, and email activity.
+sources:
+  - name: EventLogs_Exchange
+    parameters:
+      LogNames:
```
Comment (scudette) [inline content/exchange/artifacts/ExchangeServerHunting:9]: This is invalid

File: content/exchange/artifacts/ExchangeServerHunting [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        - "Application"
+        - "System"
+        - "Security"
+    query: >
+      SELECT EventID, TimeCreated, ProviderName, Message
+      FROM events
```
Comment (scudette) [inline content/exchange/artifacts/ExchangeServerHunting:15]: where do you define events?

File: content/exchange/artifacts/ExchangeServerHunting [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+             Message LIKE '%login attempt%' OR
+             Message LIKE '%unexpected error%')
+
+  - name: IISLogs_Hunting
+    parameters:
+      Paths:
+        - "C:\\inetpub\\logs\\LogFiles\\W3SVC*\\u_ex*.log"
+    query: >
+      SELECT *
+      FROM files()
```
Comment (scudette) [inline content/exchange/artifacts/ExchangeServerHunting:27]: there is not files() plugin

File: content/exchange/artifacts/ExchangeServerHunting [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+            Message LIKE '%EncodedCommand%'
+
+  - name: RegistryKeys_Hunting
+    parameters:
+      Keys:
+        - "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\ExchangeServer"
+        - "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows\\CurrentVersion\\Run"
+    query: >
+      SELECT Key, ValueName, Value
+      FROM registry()
```
Comment (scudette) [inline content/exchange/artifacts/ExchangeServerHunting:62]: there is not registry() plugin

**Pull request closed without merging**